src/lib/Components/Consignments/Screens/SelectFromPhotoLibrary.tsx

```diff
@@ -6,7 +6,7 @@ import ConsignmentBG from "../Components/ConsignmentBG"
 import ImageSelection, { ImageData } from "../Components/ImageSelection"
 import { BodyText as P } from "../Typography"
 
-import triggerCamera from "../../../NativeModules/triggerCamera"
+import { triggerCamera } from "lib/NativeModules/triggerCamera"
```

Comment:
we want to be moving away from default exports, and it was easier to write tests against as an individual export